pymc3/sampling.py

```diff
@@ -1070,6 +1083,13 @@ def sample_posterior_predictive(trace, samples=None, model=None, vars=None, size
 
     model = modelcontext(model)
 
+    if var_names is not None:
+        if vars is not None:
+            raise Exception("Should not specify both vars and var_names arguments.")
```

Comment:
`ValueError` for a bad argument here (`Exception` is too broad, I think)

Reply:
See 5e546b252ec20b5b202037de99f377da91b3356c